GDJS/Runtime/events-tools/soundtools.ts

```diff
@@ -428,6 +428,24 @@ namespace gdjs {
         runtimeScene
           .getSoundManager()
           .unloadAudio(soundFile, /* isMusic= */ true);
+
+      export const interpolateAudioTo = (
+        runtimeScene: gdjs.RuntimeScene,
+        channel: integer,
+        toVolume: float,
+        timeOfFade: float
+      ) => {
+        const sound = runtimeScene.getSoundManager().getSoundOnChannel(channel);
+        if (sound)
```

Comment:
To be safe, you can add some value checks on the inputs:
- Make sure duration is a positive float (and divide it per 1000 once the parameter is asked in seconds)
- Make sure your volume is positive, but I don't know if there should be a maximum value. Maybe just a warning that over 100% it could alter the quality? I'm not even sure about that.